packages/cli/src/main.ts

```diff
@@ -60,6 +60,14 @@ export const createCLI = () => {
       '--name <value>',
       'name of plugin (Lowercase alphanumeric and dash)',
     )
+    .option(
+      '--publish',
+      'This flag is only applied when a plugin is being updated! Publishes a new version of the plugin.',
+    )
+    .option(
+      ' --no-publish',
```

Comment:
I don't know if the extra space could be an issue.

```suggestion
      '--no-publish',
```